js/views/playingScreen.js

```diff
@@ -446,11 +391,8 @@ var PlayingScreen = createReactClass({
     return (
       <div
         className={className}
-        ref="PlayingScreen"
-        onTouchStart={this.handleTouchStart}
```

Comment:
I think this `onTouchStart` shouldn't be removed. There's a handler that sets the `hasCheckedMouseOverControls` var

Reply:
Fixed